libs/s3/s3_multipart_test_lib.py

```diff
@@ -145,7 +145,7 @@ def upload_parts(self, mpu_id: int = None, bucket_name: str = None, object_name:
             with open(multipart_obj_path, "rb") as file_pointer:
                 i = 1
                 while True:
-                    data = file_pointer.read(1048576 * single_part_size)
```

Comment:
Reading chunk from file is not depend on b_size as this is the actual size so this change will impact all tests wherever this  method being used also ur reading more than file_size in one chunk which is not correct in case of multipart upload. I would recomend, you should revert this change.

Reply:
Okay. Make sense.
I will revert the changes